<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>511</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>Allgemein</string>
    </attribute>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="portLb">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>105</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="playernameLb">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>29</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Player:</string>
     </property>
    </widget>
    <widget class="QLabel" name="languageLb">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>65</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="playernameEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>25</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="portEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>101</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="languageCb">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>61</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="volumeLb">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>143</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume:</string>
     </property>
    </widget>
    <widget class="QLabel" name="volumeIntLb">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>341</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="musicCb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="musicLb">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>190</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Music:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="carddeck">
    <attribute name="title">
     <string>Decks</string>
    </attribute>
    <widget class="QPushButton" name="carddeck1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>71</width>
       <height>96</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="carddeck2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>71</width>
       <height>96</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QRadioButton" name="carddeck1radio">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Deck 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="carddeck2radio">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Deck 2</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>341</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Special Cards</string>
    </attribute>
    <widget class="QLabel" name="draw2xlbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ziehe 2 Karte</string>
     </property>
    </widget>
    <widget class="QLabel" name="wishCardlbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>WÃ¼nsche Karte</string>
     </property>
    </widget>
    <widget class="QLabel" name="switchDirectionlbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Richtungs Wechsel Karte</string>
     </property>
    </widget>
    <widget class="QLabel" name="skipPlayerlbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Aussetzen Karte</string>
     </property>
    </widget>
    <widget class="QComboBox" name="draw2xcombo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Queen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>King</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ace</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="wishcombo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Queen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>King</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ace</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="changedirectcombo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Queen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>King</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ace</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="skipplayercombo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string comment="1">7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="2">8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="3">9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="4">10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="5">Jack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="6">Queen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="7">King</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="8">Ace</string>
      </property>
     </item>
    </widget>
    <widget class="QDialogButtonBox" name="okdiscardbtngrp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>341</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>OptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>OptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
